apps/ui/src/components/Ui/WrapperInput.vue

```diff
@@ -1,18 +1,41 @@
 <script setup lang="ts">
-const props = defineProps<{
-  definition: any;
-  error?: string;
-  dirty?: boolean;
-}>();
+const props = withDefaults(
+  defineProps<{
+    definition: any;
+    inputLength?: number;
+    error?: string;
+    dirty?: boolean;
+  }>(),
+  { inputLength: 0 }
+);
 
 const showError = computed(() => props.error && props.dirty);
 </script>
 
 <template>
   <div class="s-base" :class="showError ? 's-error' : ''">
-    <label v-if="definition.title" class="s-label" v-text="definition.title" />
+    <div class="!flex s-label w-full">
+      <label v-if="definition.title" class="grow" v-text="definition.title" />
```

Comment:
There is looks like definition.title can be empty (which actually looks ugly), but if we make it empty char counter will end up on left side instead of right.

Maybe to just be safe we should remove `v-if` here and use empty string if it's empty so layout is constant no matter if title is there or not.

Reply:
Should be fixed now.

Also handling the case where the title is very long

![Screenshot 2024-04-02 at 22 33 49](https://github.com/snapshot-labs/sx-monorepo/assets/495709/a13e194b-03ca-44bc-a940-71845662d575)

![Screenshot 2024-04-02 at 22 33 38](https://github.com/snapshot-labs/sx-monorepo/assets/495709/8568df4e-e629-4876-988b-e82c20336523)